src/read.c

```diff
@@ -1740,7 +1740,10 @@ static avifBool avifParseColourInformationBox(avifProperty * prop, const uint8_t
     AVIF_CHECK(avifROStreamRead(&s, colorType, 4));
     if (!memcmp(colorType, "rICC", 4) || !memcmp(colorType, "prof", 4)) {
         colr->hasICC = AVIF_TRUE;
-        colr->icc = avifROStreamCurrent(&s);
+        // Remember the offset of the ICC chunk relative to the beginning of the stream. A direct pointer cannot be stored because
```

Comment:
Pedantic nit: ICC chunk => "ICC payload" or "ICC profile"

HEIF/MIAF/AVIF doesn't seem to use the term "chunk" in the same meaning as some other image formats such as PNG.